packages/utilities/psammead-styles/src/colours.js

```diff
@@ -17,4 +17,7 @@ export const C_LUNAR = '#F2F2F2';
 export const C_GHOST = '#FDFDFD';
 
 // Colours from other BBC services
+export const C_CONSENT_BACKGROUND = '#323232';
```

Comment:
I assume these colours have been approved for articles use? Are they so specific to the consent banner that they cant have a UX style name?

Reply:
These are the colours that are used by the Orbit team for the standard cookie banner on Canonical pages https://github.com/bbc/nav-orbit/blob/85e4a6fe36e42e1f51e6752be6d34ef139c8422e/static-sources/sass/bbccookies.scss#L3-L7
These colours have also been used for the AMP implementation in the Alephamp codebase.
Natalie said she was okay with not giving them bespoke names, since they're BBC-wide colours for the consent banner.